src/SDKs/CognitiveServices/dataPlane/Vision/FormRecognizer/FormRecognizer/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+
+[assembly: AssemblyTitle("Microsoft Cognitive Services Form Recognizer SDK")]
+[assembly: AssemblyDescription("Provides access to the Microsoft Cognitive Services Form Recognizer APIs.")]
+
+[assembly: AssemblyVersion("0.8.0.0")]
+[assembly: AssemblyFileVersion("1.0.0.0")]
```

Comment:
`AssemblyFileVersion` must also be set to `0.8.0.0`

Reply:
done.